gauntlet/packages/gauntlet-solana-contracts/src/lib/utils.ts

```diff
@@ -7,3 +11,34 @@ export const divideIntoChunks = (arr: Array<any> | Buffer, chunkSize: number): a
   }
   return chunks
 }
+
+export const parseContractErrors = async (sendTx: Promise<String>, idl: Idl): Promise<string> => {
```

Comment:
Don't wrap the execution, wrap the function. Same as here (https://github.com/smartcontractkit/chainlink-solana/blob/develop/gauntlet/packages/gauntlet-serum-multisig/src/commands/multisig.ts#L99)

Reply:
Adding it to SolanaCommand, since it is a method and requires some configuration from this.provider.